src/meter.tsx

```diff
@@ -0,0 +1,494 @@
+// https://www.figma.com/design/jWWCJ9kYl6I5uHLz3GcgRp/%F0%9F%94%B6-%5BREADY%5D-Data-Visualizations-Specs
+'use client';
+import * as React from 'react';
+import {vars} from './skins/skin-contract.css';
+// @ts-expect-error - no types
+import bezier from 'cubic-bezier';
+import {getPrefixedDataAttributes} from './utils/dom';
+import {useThemeVariant} from './theme-variant-context';
+import {useElementDimensions, useTheme} from './hooks';
+import {meterPercentageLabel, meterSegmentLabel} from './text-tokens';
+import {isRunningAcceptanceTest} from './utils/platform';
+
+import type {DataAttributes} from './utils/types';
+
+const VIEW_BOX_WIDTH = 100;
+const CENTER_X = VIEW_BOX_WIDTH / 2;
+const CENTER_Y = VIEW_BOX_WIDTH / 2;
+
+const STROKE_WIDTH_PX = 6;
+const SEPARATION_PX = 2;
+
+const ANIMATION_DELAY_MS = 200;
+const ANIMATION_DURATION_MS = 1000;
+const ANIMATION_EPSILON = 1000 / 60 / ANIMATION_DURATION_MS / 4;
+
+const SMALL_VALUE_THRESHOLD = Math.PI / 10000;
```

Comment:
This works fine for semi-circles, but in circular meter, this value should be multiplied by 2 (360 degrees = 2*pi)

I don't think this is critical, but if you want the same threshold for both types of meter, you should take it into account

Reply:
ok, not super relevant, I just want a "small enough value". In fact this is a leftover from a previous implementation. At this moment all values are internally represented in the range [0..1], so it would be better to use a 0.0001

Reply:
If this is leftovers, maybe you can replace it with 0.0001?

Otherwise, anyone that will read this code in the future may think that using the calculation with math.PI is relevant

Reply:
done